.github/workflows/compute-incremental-coverage.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import io
+import json
+import re
+import subprocess
+import zipfile
+
+def StripGitDir(path):
+  # Strip the path to the git clone, leaving only the source path within the
+  # repo.
+  return re.sub(r'.*?/(lib|ui)/', r'\1/', path)
```

Comment:
Yes, true.  If I could make the coverage data from Karma/Babel/Istanbul use relative paths, that would be ideal, but I don't know how yet.  Since this tool runs in a workflow separate from the generation of the coverage data, I can't directly use the path to the git clone.  I could maybe store the path to the git clone inside the zip file... But anyway, I'd rather use this for now, and improve on it later.  I'll add a TODO to revisit this.

Reply:
Done.